bindings/dart/lib/src/web/worker/message/request.dart

```diff
@@ -57,14 +57,20 @@ class CreateParams implements ToJson {
   final Uint8List ltrModel;
   @Uint8ListConverter()
   final Uint8List wasmModule;
+  final String wasmScript;
   @Uint8ListMaybeNullConverter()
   Uint8List? serialized;
```

Comment:
with the latest change this field can be `final` as well
```suggestion
  final Uint8List? serialized;
```